src/index.js

```diff
@@ -147,6 +154,10 @@ export default class Luxafor {
    * @return {object} Instance
    */
   color(r = 0, g = 0, b = 0) {
+    if (invalidNums([...arguments])) {
+      throw new Error(`Specified values: ${[...arguments]} are invalid`);
+    }
```

Comment:
I think we should move this to exec() method, so it's not duplicated for each additional method.